core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -14,7 +14,11 @@
 
 import java.util.Objects;
 
+import io.vertx.core.net.ProxyType;
 import org.eclipse.hono.util.Constants;
+import org.springframework.util.StringUtils;
```

Comment:
we are currently trying to get rid of the dependency on Spring altogether in `hono-core` (see #539).
please replace references to `StringUtils` with custom checks

Reply:
ok